### ISSUE TITLE
Add logo and upgrade design

### ISSUE THREAD

**Issue by juanpicado** [verdaccio/verdaccio.github.io#1]
# Upgrade

We have a simple website and we will have to integrate the new logo as soon https://github.com/verdaccio/verdaccio/issues/328 finish and we have a winner. We have to consider the following guidelines.

- Mobile first
- Create multiple logo sizes for different resolutions
- Redesign header
- Colors should be according to the logo design

**Comment by rodriguesbreno** [comment]
![verdaccio github io](https://user-images.githubusercontent.com/19731692/32000660-2ce85342-b976-11e7-95e0-e8cea394d5db.png)

**Comment by juanpicado** [comment]
I love it !!

**Comment by rodriguesbreno** [comment]
I don't know much about GitHub pages, it seems being created using markdown.
I propose write the new page with React (more specifically using Gatsby to generate the static pages ) https://github.com/gatsbyjs/gatsby
Such design will be simple to implement (I can handle it, but will need help later with the deploy)

**Comment by juanpicado** [comment]
Give me until next week, these days I'm traveling, internet is quite limited ... 😞

**Comment by rodriguesbreno** [comment]
Sounds good, let's discuss more about the redesign before implement it.

**Comment by juanpicado** [comment]
Yes, we have a slack channel, it's much easier for discussions when we live in different time zones. https://verdaccio-npm.slack.com

**Comment by juanpicado** [comment]
Unfortunately is only under invitation, let me know whether I can use the email in your profile to send you an invitation

**Comment by rodriguesbreno** [comment]
@juanpicado Yes, you can send an invitation to the email at my profile 😄

**Comment by juanpicado** [comment]
Done
